server/docker/docker-compose_template

```diff
@@ -3,7 +3,8 @@ services:
   cpsrv:
     build:
       context: .
-      dockerfile: docker/dockerfile_cpsrv
```

Comment:
Just may be simply name it `docker/dockerfile_cpsrv` in any configuration due to each one is generated in all cases?